hls4ml/batch/do_hls_things.py

```diff
@@ -170,22 +173,39 @@ def sigmoid(x):
 
 proj_loc = out_loc_name + f'/{hls_model_name}/myproject_prj/'
 
-# fpgapart = 'xcu250-figd2104-2L-e'
-fpgapart = 'xcvu13p-fsga2577-2-e'
-
 if 'tcn' in mod_name or 'TCN' in mod_name:
     print('~~~ TCN is in the name, so will assume it is a CNN ~~~')
-    cfg = hls4ml.converters.create_config(backend='Vivado')
-    cfg['IOType']     = 'io_stream' # Must set this if using CNNs!
+    cfg = hls4ml.converters.create_config(backend='VivadoAccelerator')
+    cfg['IOType']     = 'io_stream' # Must set this if using CNNs!                                                                                                               
     cfg['HLSConfig']  = config
     cfg['KerasModel'] = model
     cfg['OutputDir']  = proj_loc
-    cfg['XilinxPart'] = fpgapart
+    cfg['ClockPeriod']= 5
+    cfg['XilinxPart'] = 'xczu9eg-ffvb1156-2-e'
```

Comment:
Define fpgapart separately as in main, it will be used again in the code.

Reply:
ok